### PR TITLE
␡ Enable strikethrough as markdownit extension

### PR DESCRIPTION
**Pull request by sglyon**
Ends up parsing out to a `delete` node and uses the rendering logic for the delete directive

### CODE REVIEW

**Comment by sglyon** [comment]
Hi @rowanc1 is this ready to merge?

**Comment by rowanc1** [comment]
Added a test - thanks for the nudge.

Will get this in when it goes green!

File: packages/myst-parser/src/myst.ts [hunk 1]
```diff
@@ -75,6 +76,7 @@ export function createTokenizer(opts?: Options) {
   if (markdownit.linkify) {
     tokenizer.linkify.tlds(tlds.filter((tld) => !EXCLUDE_TLDS.includes(tld)));
   }
+  if (extensions.strikethrough) tokenizer.enable('strikethrough');
```
Comment (rowanc1) [inline packages/myst-parser/src/myst.ts:79]: Note: This is not enabled by default and I am not sure if there is a way to turn this on from the CLI quite yet. We can add those features in future!

**Comment by rowanc1** [comment]
Thanks @sglyon for your help!

**Pull request merged**